### PR TITLE
MQE-1520: Consolidate WAIT_TIMEOUT in .env and pageload_timeout

### PR DESCRIPTION
**Pull request by KevinBKozan**
### Description
* Added WAIT_TIMEOUT to default yml file + default definition as this makes it required to be defined
* Removed some dead code since it's always defined now.

### CODE REVIEW

File: dev/tests/_bootstrap.php [hunk 1]
```diff
@@ -45,7 +45,8 @@
     'MAGENTO_BACKEND_NAME' => 'admin',
     'MAGENTO_ADMIN_USERNAME' => 'admin',
     'MAGENTO_ADMIN_PASSWORD' => 'admin123',
-    'DEFAULT_TIMEZONE' => 'America/Los_Angeles'
+    'DEFAULT_TIMEZONE' => 'America/Los_Angeles',
+    'WAIT_TIMEOUT' => '10'
```
Comment (jilu1) [inline dev/tests/_bootstrap.php]: Do we want this WAIT_TIMEOUT to be defined in .env ? or hiding from user in bootstrap?

File: dev/tests/functional/standalone_bootstrap.php [hunk 2]
```diff
@@ -49,7 +49,10 @@
 
     defined('DEFAULT_TIMEZONE') || define('DEFAULT_TIMEZONE', 'America/Los_Angeles');
     $env->setEnvironmentVariable('DEFAULT_TIMEZONE', DEFAULT_TIMEZONE);
-    
+
+    defined('WAIT_TIMEOUT') || define('WAIT_TIMEOUT', 10);
+    $env->setEnvironmentVariable('WAIT_TIMEOUT', 10);
```
Comment (jilu1) [inline dev/tests/functional/standalone_bootstrap.php]: Is 10 seconds sufficient for page load? especially when page is not cached.

File: dev/tests/_bootstrap.php (hunk 1 above)
Comment (jilu1) [reply dev/tests/_bootstrap.php]: Never mind. WAIT_TIMEOUT is in .env.example.

Comment (jilu1) [reply dev/tests/_bootstrap.php]: Let's add WAIT_TIMEOUT in Dev Doc.

Comment (KevinBKozan) [reply dev/tests/_bootstrap.php]: Yup, I'll add it to devdocs!

File: dev/tests/functional/standalone_bootstrap.php (hunk 2 above)
Comment (KevinBKozan) [reply dev/tests/functional/standalone_bootstrap.php]: Honestly I don't think so. We can change this default to 30.
I changed it to 10 as that's what it was in ActionObject, that may have been the wrong call.

**Comment by tomreece** [comment]
@jilu1 Are you sure that WAIT_TIMEOUT is required now?
it looks to me like defaults are provided but I haven’t ran it locally to confirm.

If it is in fact required, I think we should undo that and make it NOT required. We should provide a sensible default of 30 seconds or something like that.

**Comment by jilu1** [comment]
I saw it from a different perspective that WAIT_TIMEOUT is always required now but it’s Ok with me if we want to "hide" it from .env.

**Pull request merged**